deploy/charts/istio-csr/templates/clusterrole.yaml

```diff
@@ -21,3 +22,4 @@ rules:
   - "tokenreviews"
```

Comment:
Does this actually work? istio-csr requires the ability to perform a TokenReview... so by not granting this permission, it surely won't be able to do _anything_?

Reply:
The idea here is that the deployments on the cluster level, the role and the rolebinding, could be handled outside of this deployment. We are running multiple deployments of istio and istio-csr in our clusters, but these resources can only be set once, so we can't use the helmtemplates as they are right now. For a single deployment, this should always be created, which is why the default is false on the setting.